bot/cogs/fun.py

```diff
@@ -199,6 +199,43 @@ async def lizard(self, ctx) -> None:
                 else:
                     await ctx.send(f"Something went Boom! [status : {lizr.status}]")
 
+    @command(aliases=["leet"])
+    async def leetify(self, ctx: Context, *, content: str) -> None:
+        """Gives each letter of a given message a different markdown style."""
+
+        leetters = {
+            "a": ["A","4"], "e": ["E", "3"], "i": ["1", "I", "i"],
+            "o": ["O", "0", "o"], "s": ["5", "S", "s"]
+            }
+        content = [i.lower() + " " for i in content]
+        str_content = ""
+        for i in content:
+            str_content += i
+        ref_content = ""
```

Comment:
This seems like a better name for this.
Do note that you will need to update all references to `ref_content` with this new name.
```suggestion
        leetified_content = ""
```